test/woodbury.jl

```diff
@@ -65,6 +65,18 @@ for elty in (Float32, Float64, ComplexF32, ComplexF64, Int)
     @test transpose(W)*v ≈ transpose(F)*v
     @test (W + W) \ v ≈ (2*Matrix(W)) \ v
 
+    # logdet
+    # make sure all matrices are PSD because I don't want complex numbers
+    randpsd(S) = randn(S) |> Q -> Q * Q'
+    W = Woodbury(T, randpsd(size(T)), randpsd(size(T)), randpsd(size(T)))
+    @test logdet(W) ≈ logdet(W.A) + logdet(W.C) - logdet(W.Cp)
+    @test logdet(W) ≈ log(det(W))
+    lad, s = logabsdet(W)
+    @test lad ≈ logabsdet(W.A)[1] + logabsdet(W.C)[1] - logabsdet(W.Cp)[1]
+    @test s == logabsdet(W.A)[2] * logabsdet(W.C)[2] / logabsdet(W.Cp)[2]
+    @test s == sign(det(W.A)) * sign(det(W.C)) / sign(det(W.Cp))
+    @test lad ≈ log(abs(det(W)))
```

Comment:
Maybe this whole block could just be `@test logabsdet(W) ≈ logabsdet(Array(W))`?

Reply:
Thats....a good point.